operating/manage-vulnerabilities/scan-rhcos-node-host.adoc

```diff
@@ -12,7 +12,7 @@ With {rh-rhacs-first}, you can scan {op-system} nodes for vulnerabilities and de
 
 {product-title-short} scans {op-system} RPMs installed on the node host, as part of the {op-system} installation, for any known vulnerabilities.
 
-First, {product-title-short} analyzes and detects {op-system} components. Then it matches vulnerabilities for identified components by using {op-system-base} and OpenShift 4.X Open Vulnerability and Assessment Language (OVAL) v2 security data streams.
+First, {product-title-short} analyzes and detects {op-system} components. Then it matches vulnerabilities for identified components by using {op-system-base} and its security streams. Scanner V4 uses https://security.access.redhat.com/data/csaf/v2/vex/[VEX]. For more information, see xref:../../architecture/acs-architecture.adoc#con-vuln-sources_acs-architecture[Vulnerability data sources].
```

Comment:
what do `op-system` and `op-system-base` turn into?

Reply:
If you click the preview link in the body of the PR, it takes you to the HTML and you should be able to see what the variables resolve to (in this case, RHCOS and RHEL). 
![Screenshot 2024-10-25 at 10 29 59 AM](https://github.com/user-attachments/assets/cdfd8694-bd0e-4183-ba8b-472c7abf6558)

Reply:
I don't think `by using {op-system-base}` makes a lot of sense. I think the original wording was actually correct (RHEL and OpenShift 4.x streams). Perhaps you can say "Then it matches vulnerabilities for identified components by using both {op-system-base} and OpenShift 4.X Open Vulnerability and Assessment Language (OVAL) v2 security data streams."

Reply:
Oh, OK, I think I misunderstood because I thought that VEX was replacing OVAL v2 for RHCOS scanning also.